src/Workspaces/Workspaces.cs

```diff
@@ -0,0 +1,171 @@
+﻿using CoreNodeModels;
+using CoreNodeModels.Input;
+using Dynamo.Core;
+using Dynamo.Engine;
+using Dynamo.Graph.Nodes;
+using Dynamo.Graph.Nodes.CustomNodes;
+using Dynamo.Graph.Nodes.NodeLoaders;
+using Dynamo.Graph.Workspaces;
+using Dynamo.Scheduler;
+using Newtonsoft.Json;
+using PythonNodeModels;
+using System;
+using System.Collections.Generic;
+using System.Text.RegularExpressions;
+
+namespace Autodesk.Workspaces
+{
+    public static class Utilities
```

Comment:
API Change : The `Workspaces` assembly is added with methods for serializing and deserializing `WorkspaceModel` objects to json.